release/release_logs/1.9.0/benchmarks/many_actors.json

```diff
@@ -0,0 +1,10 @@
+{
+  "actors_per_second": 234.49773688626482,
```

Comment:
Hmm this is half compared to 1.9. @wuisawesome is this okay?

Reply:
Hmmm, maybe it's related to internal kv staff. I think moving from talking to redis directly to grpc introduces the regression.

Reply:
@iycheng can you comment on this actually? 1.8.0 was unusually high, but I assumed that was the actor scheduling performance. Is it bad that it's gone back down?

Reply:
At 1.7 it was 333, maybe we can take a look at the nightly dashboard to see if the regression is serious

Reply:
<img width="339" alt="Screen Shot 2021-11-29 at 5 24 31 PM" src="https://user-images.githubusercontent.com/18510752/143968762-7047f0c1-29f1-47a8-b379-e71cdb534b59.png">

Reply:
So, I think there was some regression from Redis -> GCS (30~is seconds -> 50~ish seconds)

Reply:
I think it's not related to actor schedules. We need to put kv as a separate thread.